Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Outlook.Grid/Classes/Formating/ColourFormatting.cs

```diff
@@ -82,9 +82,9 @@ public static Color ConvertTwoRange(double value, double min, double max, TwoCol
         /// <returns></returns>
         public static Color ConvertThreeRange(double value, double min, double max, ThreeColoursParams par)
         {
-            ColourHelper.HSVColour A = ColourToHSV(par.MinimumColour);
-            ColourHelper.HSVColour B = ColourToHSV(par.MaximumColour);
-            ColourHelper.HSVColour C = ColourToHSV(par.MediumColour);
+            ColourHelper.HsvColour a = ColourToHsv(par.MinimumColour);
```

Comment:
`HSV` needs to be a valid string naming type

Reply:
Done